packages/appstore/src/components/trade-button/trade-button.tsx

```diff
@@ -1,9 +1,9 @@
 import { Button } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { Actions } from 'Components/containers/trading-app-card-actions';
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { useStores } from 'Stores';
 
 const TradeButton = ({
```

Comment:
since we are using observer from deriv store, we can use observer like below.

const TradeButton = observer({.....})


...
export default TradeButton;